rasa/core/lock_store.py

```diff
@@ -79,7 +82,9 @@ def issue_ticket(
         logger.debug(f"Issuing ticket for conversation '{conversation_id}'.")
         try:
             lock = self.get_or_create_lock(conversation_id)
-            ticket = lock.issue_ticket(lock_lifetime)
+            lock.remove_expired_tickets()
+            ticket_number = self.increment_ticket_number(lock)
+            ticket = lock.issue_ticket(lock_lifetime, ticket_number)
```

Comment:
`remove_expired_tickets` happens both here and in `lock.issue_ticket` - is it needed to do twice?

Reply:
We need to have it before the call to `increment_ticket_number`, however I think it's redundant now in `lock.issue_ticket()`. I'll have a look and remove any redundant lines.